lib/jekyll/collection.rb

```diff
@@ -210,12 +210,11 @@ def container
     private
 
     def read_document(full_path)
-      doc = Jekyll::Document.new(full_path, :site => site, :collection => self)
-      doc.read
-      if site.publisher.publish?(doc) || !write?
-        docs << doc
-      else
-        Jekyll.logger.debug "Skipped Publishing:", doc.relative_path
+      docs << Document.new(full_path, :site => site, :collection => self).tap do |doc|
+        doc.read
+        if !site.publisher.publish?(doc) && site.publisher.hidden_in_the_future?(doc)
+          Jekyll.logger.debug "Skip Publishing:", "#{doc.relative_path} has a future date"
```

Comment:
I am unable to trigger this log message. Perhaps I am misunderstanding.

https://github.com/jekyll/cases/commit/0c7256b03a1c24bf9afe2ad3c45627a1e3fabd1b

Reply:
can you post the *entire* verbose build output under the commit? Thanks

Reply:
**verbose** is the key word. 🤕